submarine-server/server-submitter/submitter-k8s/src/test/resources/tf_job_mnist.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: "kubeflow.org/v1"
```

Comment:
Is this tf_job_mnist.yaml file used anywhere?

Reply:
These files are the templates of the tf-operator's mnist job. So I think we'd better keep these and it will be used in the CI.